Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN mkdir -p /usr/share/GeoIP/ && \
     mv GeoLite2-Country_20200128/*.mmdb /usr/share/GeoIP/
 
 # Upgrade package essentials.
-RUN pip3 install --upgrade pip==20.0.2 setuptools wheel dumb-init pipenv
+RUN pip3 install --upgrade setuptools wheel dumb-init pipenv
```

Comment:
The problem I see with removing specific version is compatibility. As new packages are released almost weekly, it's hard for things not to break.